test/parallel/test-timers-unrefd-interval-still-fires.js

```diff
@@ -2,19 +2,23 @@
 /*
  * This test is a regression test for joyent/node#8900.
  */
-require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
-var N = 5;
+const TEST_DURATION = common.platformTimeout(100);
+const N = 5;
 var nbIntervalFired = 0;
-var timer = setInterval(function() {
+
+const keepOpen = setTimeout(() => {
+  assert.strictEqual(nbIntervalFired, N);
```

Comment:
It's probably more informative to just throw like https://github.com/nodejs/node/blob/master/test/parallel/test-timers-unref-active-unenrolled-disposed.js#L14